blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/BlueOceanUrlAction.java

```diff
@@ -35,4 +37,8 @@ public String getUrlName() {
     BlueOceanUrlObject getBlueOceanUrlObject(){
         return blueOceanUrlObject;
     }
+
+    public boolean isAnalyticsEnabled() {
```

Comment:
Maybe refactor these to a single bit of logic to determine enabled/not? e.g. could add flags, options, etc. in the future. E.g. `Analytics.isEnabled()`

Reply:
Done